### ISSUE TITLE
Primary and Secondary colors are ignored in 2.1.2

### ISSUE THREAD

**Issue by nemchik** [atom-material/atom-material-ui#463]
I just upgraded to 2.1.2 and have found that my selected Primary and Secondary colors are no longer applied.

**Reproduction Steps:**

1. Have a Primary and Secondary color set

**Expected behavior:**

The UI should use my Primary and Secondary selected colors.

**Observed behavior:**

The UI uses the default teal color

**Screenshots and GIFs**

![Screenshots and GIFs which follow reproduction steps to demonstrate the problem](https://i.imgur.com/xz8JbnA.png)

**Atom version:** 1.24.0 x64
**OS and version:** Microsoft Windows [Version 10.0.16299.248]

**Installed packages:**

Community Packages (36) \.atom\packages
├── activate-power-mode@2.7.0
├── atom-alignment@0.13.0
├── atom-beautify@0.30.9
├── atom-bracket-highlight@0.3.2
├── atom-material-ui@2.1.2
├── auto-detect-indentation@1.3.0
├── busy-signal@1.4.3
├── duplicate-removal@0.1.3
├── file-icons@2.1.16
├── highlight-selected@0.13.1
├── indent-guide-improved@1.4.13
├── intentions@1.1.5
├── language-autohotkey2@1.0.1
├── language-batch@0.4.0
├── language-generic-config@1.4.0
├── language-htaccess@0.2.0
├── language-ini@1.19.0
├── language-nginx@0.8.0
├── language-nsis@6.11.2
├── language-powershell@4.0.0
├── language-vbscript@0.9.0
├── linter@2.2.0
├── linter-csslint@2.0.0
├── linter-htmlhint@1.4.0
├── linter-jshint@3.1.6
├── linter-jsonlint@1.3.0
├── linter-mixed-indent@1.1.1
├── linter-php@1.5.1
├── linter-ui-default@1.6.10
├── multi-cursor@2.1.5
├── pigments@0.40.2
├── sort-lines@0.18.0
├── split-diff@1.5.2
├── sync-settings@0.8.3
├── teletype@0.8.0
└── tree-view-git-status@1.4.0

**Comment by nemchik** [comment]
P.S. This was not an issue on the previous version prior to updating to 2.1.2 and I have tried completely closing Atom (and ensuring no atom.exe tasks remain) and then reopening and the issue persists.

**Comment by silvestreh** [comment]
have you tried setting your custom color again?

**Comment by nemchik** [comment]
Yes I have tried changing to other colors and then setting back to my preferred colors and this hasn't resolved it. I also tried the predefined color selection select list and none of them seem to work either.

**Comment by madeddy** [comment]
Have the same:
# https://github.com/atom-material/atom-material-ui/commit/d3629c59c0b548b5e98a2a9e99c36544effa73da

**Comment by entepe85** [comment]
I'm experiencing the same issue. I somehow managed to set a different color while using the Material Light Syntax theme but after changing that I am unable to change the default color.

**Comment by Nosenation** [comment]
Got same issue. Tried changing colors, as set, or individually. Nothing changes.

**Comment by xeho91** [comment]
I can confirm that **after computer restart it works**.
Using Windows 10 Pro N (64x).

**Comment by gregtyler** [comment]
This worked for me by just restarting Atom, not the whole PC.

**Comment by silvestreh** [comment]
I wasn't able to reproduce this one. But looks like the solution was to restart Atom (or your computer), so I'm thinking it might've been cache related? 😕 

On the upside, I did find another bug introduced in `2.1.2` which rendreded the accent color generator useless.

**Comment by nemchik** [comment]
I have both rebooted my PC and restarted Atom (as mentioned in my initial post and comments) and the issue persists for me. Is there something special I have to do in order to clear cache for Atom?

**Comment by nemchik** [comment]
Ok, I closed and reopened Atom again just now and the colors started working. Then I was prompted to update to 2.1.3 and I did this and was prompted to restart Atom. After restarting Atom the colors stopped working. I restarted again and the colors started working again. This is pretty irritating behavior.

**Comment by PeteCrighton** [comment]
Can confirm, happened to me after both updates to 2.1.2 & 2.1.3. Simply restarting Atom didn’t fix it, but choosing another predefined color and switching back to the desired color again, then restarting Atom fixed it for me.

**Comment by phlippieb** [comment]
Same issue on MacOS. I can't find a workaround. Have tried to
1. Select custom colours again
2. Select predefined colours, then do 1 (the UI doesn't even update to the selected predefined colours)
3. Restart atom, then do 2, then 1
4. Restart PC, then 2, then 1

**Comment by DominikZians** [comment]
I got the same issue (MacOS, Atom 1.24.0), but only in combination with the atom-material-syntax. When I choose an other syntax (for ex. atom-material-syntax-dark) the color changes to the choosen primary color. Changing back to the atom-material-syntax changes the color back to default.

**Comment by drakesong** [comment]
I have the same issue on Windows 10 *(Atom 1.24.0, atom-material-ui 2.1.3, atom-material-syntax 1.0.8)*. I can also confirm that only with both atom-material-ui and atom-material-syntax this error occurs. If I choose another syntax theme such as atom-material-syntax-dark, this selected accent colors work correctly.

**Comment by tumido** [comment]
Despite not using `atom-material-syntax`, I can also confirm this issue. I'm using [`material-design-syntax`](https://github.com/Nxt3/material-design-syntax/). Changing to some _other-and-not-affected_ syntax themes workaround this issue. The stock ones like Base16, One, Solarized works fine as well.

**Comment by nemchik** [comment]
I should note that I use `Base16 Tomorrow Dark` syntax and the issue of updates causing the colors to not work until a restart of Atom has been around for a LONG time before I opened this ticket. Prior to version 2.1.2 I was able to simply able to restart Atom one extra time after an update and the colors would start working again. 2.1.2 changed that and I was unable to get the colors to set correctly again until 2.1.3 showed up as available (see my comments above).

**Comment by micleb** [comment]
Same issue, my atom-material-ui is in 1.2.3

**Comment by karolfalkiewicz** [comment]
For me this issue occurs right now on specific syntax themes (like Atom Snazzy Clear). On standard one everything looks fine.

PS: No errors in dev console.

**Comment by biogeo** [comment]
Same issue here with Atom 1.19.1, atom-material-ui 2.1.3 running on Ubuntu Linux. Using syntax theme Gruvbox Plus causes the color settings to be ignored; other syntax themes, including atom-material-syntax, work normally.

**Comment by diddy0dante** [comment]
same issue here, I'm on macOS 10.14.4, Atom 1.36.0 x64 and atom-material-ui 2.1.3. It doesn't seem to matter which syntax theme is in use. I've restarted Atom and uninstalled and reinstalled atom-material-ui and the issue still persists. Predefined colors are also ignored. I don't really like the teal so this is quite a problem for me.

**Comment by shincap8** [comment]
Same issue here with Vscode, material-ui 4.11.2 running on macOS. can't change first custom colors, not even for the predefined ones.

**Comment by evelynhathaway** [comment]
@shincap8 this issue is for an Atom editor theme, which doesn't affect VS Code, and guessing by that version, you mean the React framework Material UI and not atom-material-ui.

**Comment by shincap8** [comment]
@evelynhathaway You're right, and I already solved it, I was importing ThemeProvider from the wrong library 😅. But thank you so much for taking the time.
